.github/workflows/static_code_analysis.yaml

```diff
@@ -19,21 +19,6 @@ jobs:
         run: |
           pip install ".[all]"
       - uses: pre-commit/action@v2.0.3
-      - name: mypy
```

Comment:
We also need to re-introduce all these static code analysis tools, if we want to port it to the template afterwards. otherwise we have to do the same work twice